WooCommerce/WooCommerceTests/ViewRelated/Search/Customer/CustomerSearchUICommandTests.swift

```diff
@@ -35,4 +42,26 @@ final class CustomerSearchUICommandTests: XCTestCase {
         XCTAssertEqual(cellViewModel.title, "\(customer.firstName ?? "") \(customer.lastName ?? "")")
         XCTAssertEqual(cellViewModel.subtitle, String(customer.email))
     }
+
+    func test_CustomerSearchUICommand_when_synchronizeModels_then_tracks_orderCreationCustomerSearch_event() {
+        // Given
+        let command = CustomerSearchUICommand(
+            siteID: sampleSiteID,
+            analytics: analytics) { _ in }
+
+        // When
+        command.synchronizeModels(
+            siteID: sampleSiteID,
+            keyword: "",
+            pageNumber: 1,
+            pageSize: 1,
+            onCompletion: { _ in }
+        )
+
+        // Then
+        guard let eventIndex = analyticsProvider.receivedEvents.firstIndex(of: "order_creation_customer_search") else {
+            return XCTFail("Analytics not logged")
+        }
+        XCTAssertNotNil(eventIndex)
```

Comment:
This test works well 👍 but this `XCTAssertNotNil` can't fail. It's a little more expressive like this:

```suggestion
        XCTAssert(analyticsProvider.receivedEvents.contains("order_creation_customer_search"))
```

Reply:
Oh! That's true, if passes the guard, it would never fail. Thanks for the suggestion! Changed at 2be505315a38d7b59f9e92a54867eafe3dd6907e